packages/grid_client/src/primitives/nodes.ts

```diff
@@ -333,9 +335,42 @@ class Nodes {
       });
   }
 
+  /**
+   * Sets the features of the node according to filter options.
+   *
+   * @param options - An object containing filter options to set the node's features.
+   * @returns A string array representing the node's features.
+   */
+  getFeaturesFromFilters(options: FilterOptions = {}): string[] {
+    const features: string[] = options.features || [];
```

Comment:
imo it will be better if we use Set to prevent duplication; we can return it as array